src/Grpc.AspNetCore.Server.GrpcClient/Grpc.AspNetCore.Server.GrpcClient.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
It has a dependency on the managed gRPC client. We don't nessessarily want the server to always bring in the client.

This package glues the two together. I think it would be something the metadata would bring in.